addOns/ascanrulesAlpha/src/main/java/org/zaproxy/zap/extension/ascanrulesAlpha/ForbiddenBypassScanRule.java

```diff
@@ -0,0 +1,193 @@
+                String SCHEMA = uri.getScheme();
+                List<String> upayloads =
+                        new LinkedList<String>(
+                                Arrays.asList(
+                                        new String[] {
+                                            "/%2e" + PATH,
+                                            PATH + "/.",
+                                            "/." + PATH + "/./",
+                                            PATH + "%20/",
+                                            "/%20" + PATH + "%20/",
+                                            PATH + "..;",
+                                            PATH + "?",
+                                            PATH + "??",
+                                            "/" + PATH + "//",
+                                            PATH + "/",
+                                            PATH + "/.testus",
+                                            PATH + "../app.py"
+                                        }));
+                List<String> hpayloads =
+                        new LinkedList<String>(
+                                Arrays.asList(
+                                        new String[] {
+                                            "X-Rewrite-URL: " + PATH,
+                                            "X-Original-URL: " + PATH,
+                                            "Referer: " + PATH,
+                                            "X-Custom-IP-Authorization: 127.0.0.1",
+                                            "X-Originating-IP: 127.0.0.1",
+                                            "X-Forwarded-For: 127.0.0.1",
+                                            "X-Remote-IP: 127.0.0.1",
+                                            "X-Client-IP: 127.0.0.1",
+                                            "X-Host: 127.0.0.1",
+                                            "X-Forwared-Host: 127.0.0.1"
+                                        }));
+
+                // sending payloads in URL.
+                for (String i : upayloads) {
+                    String TMP_URI = SCHEMA + "://" + HOST + "" + i;
+                    HttpMessage req_with_payload =
+                            new HttpMessage(new org.apache.commons.httpclient.URI(TMP_URI, true));
+                    sendAndReceive(req_with_payload);
+                    if (req_with_payload.getResponseHeader().getStatusCode() == HttpStatusCode.OK) {
+                        newAlert()
+                                .setConfidence(Alert.RISK_INFO)
+                                .setName(
+                                        Constant.messages.getString(
+                                                "ascanalpha.forbiddenBypass.name"))
+                                .setDescription(
+                                        Constant.messages.getString(
+                                                "ascanalpha.forbiddenBypass.desc"))
```

Comment:
This is redundant, `newAlert` already includes the name and description.

Reply:
I added single function for both alert and removed duplicates

Reply:
This comment was about the `setName` and `setDescription` not being needed.

Reply:
This is still pending.

Reply:
Do I have to remove `setName` and `setDescription` here?

Reply:
Yes, they are redundant.

Reply:
Done